tests/tests_catapult_integration.sh

```diff
@@ -42,7 +42,7 @@ tearDown() {
 
 testKind() {
     rm -rf buildtest
-    make kind
+    BACKEND=kind make k8s
```

Comment:
no buildir is created by the target, that's why travis complains with:
`mv: cannot move 'kind' to 'bin/kind': No such file or directory

Makefile:9: recipe for target 'deps' failed

make[3]: *** [deps] Error 1

make[3]: Leaving directory '/home/travis/build/SUSE/catapult/backend/kind'

Makefile:24: recipe for target 'k8s' failed`